java/src/jmri/jmrix/loconet/messageinterp/LocoNetMessageInterpret.java

```diff
@@ -4036,7 +4035,6 @@ private static String interpretOpcImmPacket(LocoNetMessage l) {
                 if ((nmraSubInstructionType == 0x1d) && (packetInt[3] == 0x7f)) {
                     playableWhistleLevel = packetInt[4];
                 }
-            } else {
                 // immediate packet not addressed to a multi-function (mobile) decoder
             }
```

Comment:
It seems to me that the comment should be removed as well. By removing the else statement, developers would think that the comment belongs to the if statement, not the else statement.